docs/markdown/Android/sheet.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Modal (Sheet)
+description: Sheets are bottom-anchored overlays that that allow the user to easily return to the previous screen. They are meant for temporary focused tasks. Sheets are the mobile equivalent of the modal or overlay on web.
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/41/a6/36/41a6369d262d24bf04d08ee32beef508.png" />
+
+## Usage guidelines
+
+<TwoCol>
+  <Group>
+    <Do title="When to use" />
+      - Performing an optional sub-task within a larger task
+      - Presenting help info while maintaining the current page and its context
+      - Requesting minimal amounts of information froma user (1-2 fields only)
+      - Capturing user’s full attention for something important
+  </Group>
+  <Group>
+    <Dont title="When not to use" />
+      - Displaying system errors or notices. Consider a [Callout](web/callout) instead
+      - Any time a separate, designated URL is desired
+      - Any action that should not interrupt users from their current work stream
+      - On top of another sheet, as this can create usability issues and confusion
+  </Group>
+</TwoCol>
+
+## Mobile best practices
+
+- Sheets should be triggered by the users action, like tapping on a button.
+- Use a sheet for non-imersive content and simple tasks.
+- Sheets always sit in front of other UI elements.
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/84/d4/0d/84d40d290128608db361a276a586da87.png" alt="a partial sheet with a close button and a grabber to allow for multiple ways to close and collapse"
```

Comment:
@hectoid  - could you check the `ImgContainers` with the do's. They are missing closing tags